docs/data/tree-view/rich-tree-view/editing/EditWithIcons.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { RichTreeView } from '@mui/x-tree-view/RichTreeView';
+import { useTreeItem2Utils } from '@mui/x-tree-view/hooks';
+import {
+  TreeItem2,
+  TreeItem2Label,
+  TreeItem2Props,
+} from '@mui/x-tree-view/TreeItem2';
+import { IconButton } from '@mui/material';
+import EditOutlinedIcon from '@mui/icons-material/EditOutlined';
+import CloseRoundedIcon from '@mui/icons-material/CloseRounded';
+import CheckIcon from '@mui/icons-material/Check';
+import { TreeItem2LabelInput } from '@mui/x-tree-view/TreeItem2/TreeItem2';
```

Comment:
```suggestion
import { TreeItem2LabelInput } from '@mui/x-tree-view/TreeItem2';
```

No depth 2 import :angel:

Reply:
https://github.com/mui/mui-x/pull/13401 covers this but is blocked :/
I'll try extracting this part

Reply:
whoa that's a big oopsie. Thanks for noticing it 🙈 I should check the auto imports more carefully 🙈